### PR TITLE
Create Account Module scaling (fonts and sizes able to adjust depending on device)

### PR DESCRIPTION
**Pull request by amanalili**

### CODE REVIEW

File: packages/edge-login-ui-rn/src/common/constants/Theme.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import { Platform } from 'react-native'
 
 export const HEADER_HEIGHT = Platform.OS === 'ios' ? 42 : 60
+export const HEADER_HEIGHT_1 = Platform.OS === 'ios' ? 46 : 66
```
Comment (paullinator) [inline packages/edge-login-ui-rn/src/common/constants/Theme.js]: Where is this used?

Comment (amanalili) [reply packages/edge-login-ui-rn/src/common/constants/Theme.js]: On the headers, somehow to have a uniform spacing I used that on the create account headers. I don't want to change a default that may be used by other components

File: packages/edge-login-ui-rn/src/native/components/screens/newAccount/NewAccountReviewScreenComponent.js [hunk 2]
```diff
@@ -32,14 +32,12 @@ export default class NewAccountReviewScreenComponent extends Component<Props> {
                 message={s.strings.warning_message}
               />
             </View>
-            <View style={NewAccountReviewScreenStyle.shim} />
```
[the remark this answers is not in the document]
Comment (amanalili) [reply packages/edge-login-ui-rn/src/native/components/screens/newAccount/NewAccountReviewScreenComponent.js]: Replaced it with margins

File: packages/edge-login-ui-rn/src/native/components/abSpecific/PasswordStatusComponent.js [hunk 3]
```diff
@@ -46,7 +46,6 @@ export default class PaswordStatusComponent extends Component<Props> {
       return (
         <View style={style.container}>
           <View style={style.boxes}>{this.renderStatusList(style)}</View>
-          <View style={style.shim} />
```
[the remark this answers is not in the document]
Comment (amanalili) [reply packages/edge-login-ui-rn/src/native/components/abSpecific/PasswordStatusComponent.js]: Replaced it with margins

File: packages/edge-login-ui-rn/src/common/constants/Theme.js (hunk 1 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/common/constants/Theme.js]: Ok my bad, I thought I did use it somehow, forgot to remove it, thanks.

**Pull request merged**